google/ggrc-core
# GGRC-7800 [FE] Date and date.time. Calendar widget has two variants of picked date.

# Issue description
Date and date.time. Calendar widget has two variants of picked date.

# Steps to test the changes 
1. Try to create any object with start/end date:
2. **First option**: open calendar and choose date - today > save&close. Look at request payload in devtools - there are only date(for current user timezone) was send to BE.
**Second option**: do not open calendar(by default date = today) > save&close. Look at request payload in devtools - there are date.time(UTC) was send to BE.
**Actual Result**: FE send date to BE in with time.  
**Expected result**: FE send date to BE without time.  

# Solution description 
Add formatting of date for new date instances.
Refactor datepicker component. Fix datepicker bugs.

# Sanity checklist 

# PR Review checklist
